lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1515,11 +1515,10 @@ void ResourceMap::loadDefaultConfiguration()
   addAsScalar("RandomWalkMetropolisHastings-DefaultAdaptationLowerBound", 0.117); // = 0.5 * 0.234
   addAsScalar("RandomWalkMetropolisHastings-DefaultAdaptationShrinkFactor", 0.8);
   addAsScalar("RandomWalkMetropolisHastings-DefaultAdaptationUpperBound", 0.468); // = 2.0 * 0.234
-  addAsUnsignedInteger("RandomWalkMetropolisHastings-DefaultAdaptationPeriod", 100);
+  addAsUnsignedInteger("RandomWalkMetropolisHastings-DefaultAdaptationPeriod", 30);
```

Comment:
- Why is the magic 30 number better than the magic 100 number? What is the default value in other software libraries? Is there a reference paper stating that 30 is best?
- I can be wrong, but "period" may not be the good word here. "IterationNumber"?

Reply:
Ok. So I suggest to resolve this discussion then.